content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md

```diff
@@ -798,53 +798,100 @@ updates:
 
 ### `versioning-strategy`
 
-When {% data variables.product.prodname_dependabot %} edits a manifest file to update a version, it uses the following overall strategies:
+When {% data variables.product.prodname_dependabot %} edits a manifest file to update a version, there are several different potential _versioning-strategies_:
 
-- For apps, the version requirements are increased, for example: npm, pip and Composer.
-- For libraries, the range of versions is widened, for example: Bundler and Cargo.
+| Option | Supported by | Action |
+|--------|--------------|--------|
+| `auto` | Try to differentiate between between apps and libraries. Use `increase` for apps and `widen` for libraries.|
+| `increase`| Always increase the minimum version requirement to match the new version. If a range already exists, typically this only increases the lower bound. |
+| `increase-if-necessary` | Leave the constraint if the original constraint allows the new version, otherwise, bump the constraint. |
+| `lockfile-only` | Only create pull requests to update lockfiles. Ignore any new versions that would require package manifest changes. |
+| `widen`| Widen the allowed version requirements to include both the new and old versions, when possible. Typically this only increases the maximum allowed version requirement. |
+| N/A | Some package managers do not yet support configuring the `versioning-strategy` parameter. |
+
+Example:
```

Comment:
Following our content model:
```suggestion
The following table shows and example of how `versioning-strategy` can be used.
```

Reply:
👍 but let me fix the typo of `"shows and example"` 😁

Reply:
Argh, so sorry about that 🙈